beyond-essence/inventory/toggle-switches.md

```diff
@@ -6,77 +6,63 @@ background_image: index.jpg
 ---
 {% image_custom image="/assets/img/pages/essence/switch-button-on-off.png" width="25" align="right" %}
```

Comment:
As the "Track stock level" example is outdated, I guess we shouldn't use it as an example?

Reply:
Good catch! Will replace the image.